core/psd/imx6ull/flashmng.c

```diff
@@ -102,7 +102,8 @@ int flashmng_writedev(oid_t oid, uint32_t paddr, void *data, int size, int type)
 }
 
 
-int flashmng_eraseBlock(oid_t oid, int start, int end)
+// start and end are in "erase blocks"
```

Comment:
Use C style comments (i.e /* comment */)

Reply:
uch, You mean C89-style, because these are C99-compliant :).

I'll change them.